pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.2.2</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>org.example</groupId>
+    <artifactId>Notification_demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Notification_demo</name>
+    <description>Notification_demo</description>
+    <properties>
```

Comment:
Line 11-15 add meaningful values.

<groupId>org.pigeon</groupId>
<artifactId>notification-api</artifactId>

change name and description accordingly.